UnityProject/Assets/Scripts/Effects/FloatingEffect.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FloatingEffect : LTEffect
+{
+	public bool willAnimate;
```

Comment:
do we need to read this from anywhere else?

Reply:
yes